app/lib/theme/AnticipatedFundingPerFiscalYearArrayFieldTemplate.tsx

```diff
@@ -0,0 +1,91 @@
+import { ArrayFieldTemplateProps } from "@rjsf/core";
```

Comment:
I ~*think we need to use an `ArrayFieldTemplate` because [array types can use custom widgets for individual fields](https://react-jsonschema-form.readthedocs.io/en/latest/usage/arrays/#:~:text=document.getElementById(%22app%22))%3B-,Custom%20widgets,-%C2%B6) but entire-array widgets [don't seem to be suppported](https://github.com/rjsf-team/react-jsonschema-form/issues/573#:~:text=The%20current%20ArrayField%20implementation%20doesn%27t%20support%20a%20custom%20widget)